java/compaction-status-store/src/test/java/sleeper/compaction/status/testutils/DynamoDBCompactionTaskStatusStoreTestBase.java

```diff
@@ -52,35 +52,56 @@ public void tearDown() {
         dynamoDBClient.deleteTable(taskStatusTableName);
     }
 
-    protected static Instant defaultStartTime() {
+    private static Instant defaultTaskStartTime() {
         return Instant.parse("2022-09-22T12:30:00.000Z");
     }
 
-    protected static Instant defaultFinishTime() {
+    private static Instant defaultJobStartTime() {
+        return Instant.parse("2022-09-22T14:00:04.000Z");
+    }
+
+    private static Instant defaultJobFinishTime() {
         return Instant.parse("2022-09-22T16:30:00.000Z");
     }
 
+    private static Instant defaultJobFinishTimeWithDurationInSecondsNotAWholeNumber() {
+        return Instant.parse("2022-09-22T14:00:14.500Z");
+    }
```

Comment:
This should be named defaultTaskFinishTimeWith... since it's for the task not the job.